pkg/util/util.go

```diff
@@ -44,119 +44,58 @@ func OnlyOneNotNil(items ...interface{}) (any bool, one bool) {
 	return oneNotNil, oneNotNil
 }
 
+// Deprecated: use slices#Equal instead
 func EqualSorted(s1, s2 []string) bool {
+	return slices.Equal(s1, s2)
+}
+
+func genericEqualSorted[S ~[]E, E any](
```

Comment:
https://go.dev/blog/intro-generics#:~:text=Because%20the%20empty%20interface%20is%20common%20in%20type%20parameter%20lists%2C%20and%20in%20ordinary%20Go%20code%20for%20that%20matter%2C%20Go%201.18%20introduces%20a%20new%20predeclared%20identifier%20any%20as%20an%20alias%20for%20the%20empty%20interface%20type.%20With%20that%2C%20we%20arrive%20at%20this%20idiomatic%20code%3A

Reply:
I updated the function name and added a comment.